transifex/push.py

```diff
@@ -25,18 +25,28 @@
 MESSAGE = 'Update translation strings'
 
 
-def push(clone_url, repo_owner, merge_method=DEFAULT_MERGE_METHOD):
+def push(clone_url, repo_owner, merge_method=DEFAULT_MERGE_METHOD, no_commit=False):
     """Extracts translations for the given repo, commits them, pushes them to GitHub, opens a PR, waits for status
         checks to pass, merges the PR, deletes the branch, and pushes the updated translation files to Transifex.
     """
     with repo_context(clone_url, repo_owner, BRANCH_NAME, MESSAGE, merge_method=merge_method) as repo:
         logger.info('Extracting translations for [%s].', repo.name)
         repo.extract_translations()
-        repo.commit_push_and_open_pr()
 
-        if repo.pr and repo.merge_pr() and repo.pr.is_merged():
-            logger.info('Pushing translations to Transifex for [%s].', repo.name)
-            repo.push_translations()
+        if not no_commit:
+            repo.commit_push_and_open_pr()
+            if repo.pr and repo.merge_pr() and repo.pr.is_merged():
+                push_translations_to_transifex(repo)
+        else:
+            push_translations_to_transifex(repo)
+
+
+def push_translations_to_transifex(repo):
+    """
+    Calls the push translations make target to push new translation strings to Transifex.
+    """
+    logger.info('Pushing translations to Transifex for [%s].', repo.name)
```

Comment:
Could you just add this log call to the `push_translations` method defined on `Repo`?